SPEC.md

```diff
@@ -4443,7 +4396,7 @@ task single_return_code {
   >>>
 
   runtime {
-    return_codes: 1
```

Comment:
For all of these, we want to keep `return_codes`.

Reply:
@vsmalladi oh I'm so sorry, I thought this was the WDL 1.2 one when I commented that. I was wrong in this comment, all of the WDL 1.1 references should be `returnCodes`.